canary.json

```diff
@@ -0,0 +1,7 @@
+{
+  "alpha": {"enabled": false, "waves": 1, "interval": 1},
+  "beta": {"enabled": false, "waves": 1, "interval": 1},
+  "early-access": {"enabled": true, "waves": 6, "interval": 10},
```

Comment:
1. It looks like 10 minutes may not be enough to react to release problems.
2. What do you think about using the `time.Duration` Golang format as a value for the interval?

Reply:
makes sense